core/market/tests/test_agreement.rs

```diff
@@ -17,7 +15,7 @@ use ya_service_bus::RpcEndpoint;
 const REQ_NAME: &str = "Node-1";
 const PROV_NAME: &str = "Node-2";
 
-#[cfg_attr(not(feature = "market-test-suite"), ignore)]
+#[cfg_attr(not(feature = "test-suite"), ignore)]
```

Comment:
It's still `market-test-suite` I think.

Reply:
https://github.com/golemfactory/yagna/pull/743/files#diff-e53eb5977ab5b119b4c67fe69674a9c388fa07ebcabb5009639f5daaab8e1501R9

I've changed it